rackcity/api/urls.py

```diff
@@ -52,8 +52,8 @@
     path('users/pages', views.user_page_count),
     path('users/grant-admin', views.user_grant_admin),
     path('users/revoke-admin', views.user_revoke_admin),
-    path('users/groups/get', views.user_get_groups),
-    path('users/groups/set', views.user_set_groups),
+    path('users/permissions/get', views.user_get_groups),
+    path('users/permissions/set', views.user_set_groups),
```

Comment:
changed these endpoint names because the same endpoint is used to set asset datacenter-level permissions